package.json

```diff
@@ -26,7 +26,7 @@
     "src"
   ],
   "engines": {
-    "node": ">=18.16"
+    "node": ">=18"
```

Comment:
Doesn't this include also 19, 20?

Reply:
It will. This change will allow the package to be added to projects using Node versions between 18.0 and 18.16.

Reply:
Ok thanks, yes, got it now

Reply:
That's right, by "relax" it means that the new requirement is less strict that it used to be. This change is to support an application that is currently still on Node 18.12.